acmd_test.go

```diff
@@ -155,12 +166,61 @@ func TestRunner_suggestCommand(t *testing.T) {
 			Args:   tc.args,
 			Output: buf,
 		})
-		if err := r.Run(); err == nil {
-			t.Fatal()
+		if err := r.Run(); err != nil && !strings.Contains(err.Error(), "no such command") {
+			t.Fatal(err)
 		}
 
 		if got := buf.String(); got != tc.want {
-			t.Logf("want %q got %q", tc.want, got)
+			t.Fatalf("want %q got %q", tc.want, got)
 		}
 	}
 }
+
+func TestRunner(t *testing.T) {
+	r := RunnerOf([]Command{
+		{
+			Name:        "test",
+			Description: "some test command",
+			// Do: func(ctx context.Context, args []string) error {
+			// 	return nil
+			// },
+			subcommands: []Command{
+				{
+					Name: "foo",
+					// Do: func(ctx context.Context, args []string) error {
+					// 	fmt.Fprint(os.Stderr, "foo")
+					// 	return nil
+					// },
+					subcommands: []Command{
+						{
+							Name: "for", Do: func(ctx context.Context, args []string) error {
+								fmt.Fprint(os.Stderr, "for")
+								return nil
+							},
+						},
+					},
+				},
+				{Name: "bar", Do: func(ctx context.Context, args []string) error {
```

Comment:
looks like formatting is broken

Reply:
Whole test should not be committed (at least in this way) :)